bind/gengo.go

```diff
@@ -111,6 +114,28 @@ func cgopy_decref(ptr unsafe.Pointer) {
 	refs.Unlock()
 }
 
+//export cgo_check
```

Comment:
and instead of creating a `cgo_check` function that is then cgo-exported to be called from the `gencpy` step, generate an `init()` function that will *only* be inserted inside the generated code when the Go toolchain generating that file is `>=1.6`.

this minimizes the number of call sites to maintain for consistency. (and slightly simplifies the call chain.)